server/src/graphql/resolvers/mutations.js

```diff
@@ -57,19 +57,17 @@ module.exports = {
 
   deleteWell: async (_, { id }) => {
     try {
-      await db.Well.destroy({ where: { id } });
-      return true;
+      const wellDeleted = await db.Well.destroy({ where: { id } });
+      return wellDeleted ? true : false;
```

Comment:
this ternary will never return false. If something went wrong with the deletion, the try block will catch the error.

Reply:
maybe this can just return an id or just return true?